### ISSUE TITLE
Toggle is not visible in Windows high contrast mode using Firefox

### ISSUE THREAD

**Issue by cfong613** [carbon-design-system/carbon#7278]
## Environment

> Operating system

Windows 10

> Browser

Firefox 82.0.2 (64-bit)

## Detailed description

Toggle is not visible in Windows 10 high contrast mode using Firefox

> What version of the Carbon Design System are you using?

The latest since it is reproducible in the Carbon storybook

> What did you expect to happen?

Toggle is visible

> What happened instead?

The toggle is not visible

> What WCAG 2.1 checkpoint does the issue violate?

Success Criterion 1.4.11 Non-text Contrast https://www.w3.org/TR/WCAG21/#non-text-contrast

## Steps to reproduce the issue

1. On your Windows machine, turn on High Contrast Black theme
2. Launch Firefox 
3. Navigate to https://react.carbondesignsystem.com/?path=/story/toggle--toggled
BUG:
Toggle is not visible

> Please create a reduced test case in CodeSandbox
https://codesandbox.io/s/fragrant-field-yxmrb?file=/src/index.js

## Additional information

The toggle is visible in MS Edge in high contrast

- Screenshots or code

![Screen Shot 2020-11-11 at 3 22 48 PM](https://user-images.githubusercontent.com/73359170/98860611-f35eb600-2431-11eb-8645-ce194e2b9599.png)
![Screen Shot 2020-11-11 at 3 22 18 PM](https://user-images.githubusercontent.com/73359170/98860665-06718600-2432-11eb-8350-7d2700c9d279.png)

- Notes

**Comment by tw15egan** [comment]
This was recently added in the latest version of Carbon

https://github.com/carbon-design-system/carbon/blob/97df410901197d18b504685116880c5d67b757be/packages/components/src/components/toggle/_toggle.scss#L296-L301

Keep in mind, for proper HCM in Firefox you will need to enable `prefers-contrast` 

![image (1)](https://user-images.githubusercontent.com/11928039/99004123-8b70a400-24f3-11eb-9f06-9368b25b4e90.png)
